.gitignore

```
@@ -86,4 +86,7 @@ target/
 .vscode/
 
 #Ipython Notebook
-.ipynb_checkpoints
\ No newline at end of file
+.ipynb_checkpoints
+.devcontainer/devcontainer.json
+erobs_minimal_fork.code-workspace
+.devcontainer/devcontainer.json
```

```suggestion
*.code-workspace
```
The dev container is meant to be version controlled.  Make the worskapce/vscode files generic.

---

copied.